### DISCUSSION TITLE
Tasmota send MQTT to several Domoticz servers

### DISCUSSION THREAD

**Discussion by HeinvdW** [arendst/Tasmota#20072] [General]
Is it possible to send MQTT data to several Domoticz servers.
In the Tasmota Configure -> Domoticz it's possible to put an IDX, however the other Domoticz instance has another IDX for the same switch button.
![sonoff](https://github.com/arendst/Tasmota/assets/25273989/6b3e6abd-6411-40bb-8ce9-a74e6e410910)

**Comment by sfromis** [comment]
Tasmota is not "sending" to any Domoticz server, it is publishing payloads which any other MQTT client, like multiple Domoticz can subscribe to. Works best if you can harmonize usage of the same IDX across them. If not, you could probably set up rules to publish extra payloads.

**Comment by HeinvdW** [comment]
Thanks for your answer.
Indeed publush. 😳 I'm kind of a layman. 🥴
Harmonizing IDX numbers in Domoticz seems to be impossible.

Could you give me a hint on how to set up rules to publish extra payloads from within Tasmota?

**Reply by sfromis** [reply]
Rules docs have a number of examples using publish. 
https://tasmota.github.io/docs/Rules/

**Comment by arendst** [comment]
Just for my understanding, why would you want to send messages to different domoticz servers? Are you aware of Domoticz Remote Server capabilities?

**Comment by HeinvdW** [comment]
There's the meter cupboard with smart meter and electric board to connect to and there's the living room that loves to have a better RFXtfx 433 connection.
I have been experimenting with Sonoff and ESP devices to test and communicate over MQTT.
The solution you presented using Domoticz Remote Server is just what I need and wasn't aware of.
Thanks for your help.
